Marlin/src/module/settings.cpp

```diff
@@ -1580,7 +1580,7 @@ void MarlinSettings::postprocess() {
 
         #if ENABLED(MESH_BED_LEVELING)
           if (!validating) mbl.z_offset = dummyf;
-          if (mesh_num_x == GRID_MAX_POINTS_X && mesh_num_y == GRID_MAX_POINTS_Y) {
+          if (mesh_num_x == (GRID_MAX_POINTS_X) && mesh_num_y == (GRID_MAX_POINTS_Y)) {
```

Comment:
Why the parenthesis? Shouldn’t the paren be part of the macro?

Reply:
Users are encouraged to edit these macros.

Reply:
Ah, okay. I suppose that makes sense.